### ISSUE TITLE
DenseNet misclassifies an elephant picture while VGG16 classifies it correctly

### ISSUE THREAD

**Issue by rthothad** [keras-team/keras#10487]
I tried to predict an elephant picture with DenseNet121 and DenseNet169. Both returned incorrect results and moreover both predicted completely different things. On the other hand VGG16's prediction was correct. I am not sure if I am doing anything incorrectly. Please help.

I am using keras 2.2.0.

---------------------------------------------------------------------
The code for DenseNet121 is

model = DenseNet121(input_shape=(224, 224, 3))

img_path = 'elephant.jpg'
img = image.load_img(img_path, target_size=(224, 224))
x = image.img_to_array(img)
x = np.expand_dims(x, axis=0)
x = preprocess_input(x)
print('Input image shape:', x.shape)

preds = model.predict(x)
print('Preds shape is ', preds.shape)
print('Predicted:', decode_predictions(preds)) 

This outputs:
Input image shape: (1, 224, 224, 3)
Preds shape is  (1, 1000)
Predicted: [[('n06359193', 'web_site', 0.6056899), ('n02916936', 'bulletproof_vest', 0.29209805), ('n03720891', 'maraca', 0.037738036), ('n03710637', 'maillot', 0.02146188), ('n03180011', 'desktop_computer', 0.017136803)]]
---------------------------------------------------------------------
The code for DenseNet169 is
model = DenseNet169()

img_path = 'elephant.jpg'
img = image.load_img(img_path, target_size=(224, 224))
x = image.img_to_array(img)
x = np.expand_dims(x, axis=0)
x = preprocess_input(x)
print('Input image shape:', x.shape)

preds = model.predict(x)
print('Preds shape is ', preds.shape)
print('Predicted:', decode_predictions(preds)) 

This outputs:
Input image shape: (1, 224, 224, 3)
Preds shape is  (1, 1000)
Predicted: [[('n02974003', 'car_wheel', 0.53575313), ('n04311004', 'steel_arch_bridge', 0.25891325), ('n01930112', 'nematode', 0.12976533), ('n09229709', 'bubble', 0.056118153), ('n03197337', 'digital_watch', 0.014126782)]]
---------------------------------------------------------------------
The code for VGG16 is 
from keras.applications.vgg16 import VGG16
model = VGG16()

img_path = 'elephant.jpg'
img = image.load_img(img_path, target_size=(224, 224))
x = image.img_to_array(img)
x = np.expand_dims(x, axis=0)
x = preprocess_input(x)
print('Input image shape:', x.shape)

preds = model.predict(x)
print('Preds shape is ', preds.shape)
print('Predicted:', decode_predictions(preds))

This outputs:
Input image shape: (1, 224, 224, 3)
Preds shape is  (1, 1000)
Predicted: [[('n01871265', 'tusker', 0.737143), ('n02504458', 'African_elephant', 0.24346027), ('n02504013', 'Indian_elephant', 0.01939505), ('n02408429', 'water_buffalo', 1.1600612e-06), ('n02410509', 'bison', 4.9733234e-07)]]
---------------------------------------------------------------------

**Comment by Tal-Golan** [comment]
Same issue here, using a larger sample of imagenet images.
